### PR TITLE
{chem}[intel/2019b] MDTraj v1.9.3 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/6a30a9c4754f400a84284cf68282c328 for a full test report.

File: easybuild/easyconfigs/m/MDTraj/MDTraj-1.9.3-intel-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'MDTraj'
+version = '1.9.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://mdtraj.org'
```
Comment (verdurin) [inline easybuild/easyconfigs/m/MDTraj/MDTraj-1.9.3-intel-2019b-Python-3.7.4.eb]: @boegel can this be `https`?

**Comment by verdurin** [comment]
@boegel also please supply a test build (thought I've already beaten you to it on `generoso`...)

File: easybuild/easyconfigs/m/MDTraj/MDTraj-1.9.3-intel-2019b-Python-3.7.4.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/m/MDTraj/MDTraj-1.9.3-intel-2019b-Python-3.7.4.eb]: https://mdtraj.org doesn't seem to work for me, hangs?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/092695e3b7c4311dc21a58ae9ae65ab3 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**